asteroidsyncserviced/platforms/ubuntutouch/ubuntuplatform.cpp

```diff
@@ -82,18 +95,137 @@ QString UbuntuPlatform::encodeAppName(const QString appName) const
         return "unkown";
 }
 
-void UbuntuPlatform::updateMusicStatus()
+void UbuntuPlatform::onMediaServiceReady()
+{
+    setupMusicService();
+}
+
+void UbuntuPlatform::onPreviousMusicTitle()
+{
+    return sendMusicControlCommand("Previous");
+}
+
+void UbuntuPlatform::onNextMusicTitle()
+{
+    sendMusicControlCommand("Next");
+}
+
+void UbuntuPlatform::onPlayMusicTitle()
+{
+    sendMusicControlCommand("PlayPause");
```

Comment:
Are there different commands available for Play and Pause? I can see this going out of sync easily.

Reply:
For play / pause it is the same command.
I think this is not going out of sync, because i read back the actual playing status.

Reply:
Those commands come from the bluetooth network and you never know what's going on on the other side.
Please always prefer stateless protocols https://en.wikipedia.org/wiki/Stateless_protocol where a command always does the same thing.

Reply:
If i understand you right it is better to use onPlayMusicTitle only command "Play"?

Reply:
yes, and Pause for the equivalent pause slot

Reply:
Done.
Many thanks for the hint and the patience.